.github/workflows/acceptance.yml

```diff
@@ -314,3 +313,26 @@ jobs:
           api-token: ${{ secrets.VISUAL_SNAPSHOT_SECRET }}
           gcs-bucket: 'sentry-visual-snapshots'
           gcp-service-account-key: ${{ secrets.SNAPSHOT_GOOGLE_SERVICE_ACCOUNT_KEY }}
+
+  # This check runs once all dependant jobs have passed
+  # It symbolizes that all required Acceptance checks have succesfully passed
+  # This check is the only required Github check
+  acceptance-required-check:
+    needs: [visual-diff]
```

Comment:
I don't think this will work that well. You'll still need Visual Snaphots as a required job because it can still fail (e.g. snapshot changes), but `visual-diff` will run fine because it triggers the diffing.

My line of thinking to have a separate Acceptance test "check" was in case people don't associate Visual Snapshots w/ acceptance tests. e.g. they scan only for "required" checks that have failed, but only see "Visual Snapshots" -- but this may not be the case anyway since people have asked about failing non-required checks in the past (e.g. during py3 migration).

I say try only having Visual Snapshots required like you initially suggested and go from there.